### PR TITLE
#5 Add NGXS state manager to web app

### PR DESCRIPTION
**Pull request by Art-mimiron**
NGXS #5 init + toggle loading state action

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -5,14 +5,24 @@ import { AppRoutingModule } from './app-routing.module';
...
   imports: [
     BrowserModule,
     AppRoutingModule,
-    BrowserAnimationsModule
+    BrowserAnimationsModule,
+    NgxsModule.forRoot([
+      AppState,
+    ]),
+    NgxsReduxDevtoolsPluginModule.forRoot(),
+    NgxsLoggerPluginModule.forRoot()
```
Comment (dmitryy90) [inline src/app/app.module.ts:25]: lets disable logger for prod mode

    NgxsLoggerPluginModule.forRoot({
      disabled: environment.production
    })

File: src/app/app.module.ts [hunk 2]
```diff
@@ -5,14 +5,24 @@ import { AppRoutingModule } from './app-routing.module';
...
   ],
   imports: [
     BrowserModule,
     AppRoutingModule,
-    BrowserAnimationsModule
+    BrowserAnimationsModule,
+    NgxsModule.forRoot([
+      AppState,
+    ]),
+    NgxsReduxDevtoolsPluginModule.forRoot(),
```
Comment (dmitryy90) [inline src/app/app.module.ts:24]: is it added on purpose? 
I believe we need to make sure everyone uses Ngxs Redux Devtools

File: src/app/shared/app.actions.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export class ToggleLoading {
```
Comment (dmitryy90) [inline src/app/shared/app.actions.ts:1]: try to add payload just for test

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

**Review by DmyMi — CHANGES_REQUESTED** [review]
Please use the following [template](https://github.com/ita-social-projects/OoS-Frontend/blob/main/.github/PULL_REQUEST_TEMPLATE/develop.md) for pull request message :)

File: src/app/app.module.ts (hunk 1 above)
Comment (Art-mimiron) [reply src/app/app.module.ts]: logger and devtools disabled for prod

**Review by dmitryy90 — APPROVED** [review]

**Review by DmyMi — APPROVED** [review]

**Pull request merged**